examples/one_dimensional_diffusion.jl

```diff
@@ -35,8 +35,14 @@ using Oceananigans
 # Below, we build a regular rectilinear grid with 128 grid points in the `z`-direction,
 # where `z` spans from `z = -0.5` to `z = 0.5`,
 
-grid = RegularRectilinearGrid(size=(1, 1, 128), x=(0, 1), y=(0, 1), z=(-0.5, 0.5))
+grid = RegularRectilinearGrid(size=(128), z=(-0.5, 0.5), topology=(Flat, Flat, Bounded))
 
+# The default topology is `(Periodic, Periodic, Bounded)`` but since we only want to solve
```

Comment:
```suggestion
# The default topology is `(Periodic, Periodic, Bounded)` but since we only want to solve
```

This should fix this formatting

![image](https://user-images.githubusercontent.com/20099589/113171071-294fdb80-9215-11eb-8523-9d767b8d1d17.png)